README.md

```diff
@@ -201,7 +200,7 @@ data = yf.download(  # or pdr.get_data_yahoo(...
         # fetch data by interval (including intraday if period < 60 days)
         # valid intervals: 1m,2m,5m,15m,30m,60m,90m,1h,1d,5d,1wk,1mo,3mo
         # (optional, default is '1d')
-        interval = "1m",
+        interval = "5d",
```

Comment:
The issue was that "1m" produces an error, not if it is a sensible param or not. "5d" does not produce an error and is a supported and documented error. So I'd leave it like this. Otherwise, I'd rather suggest to remove the 5d option altogether from the library.